app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:

```

Comment:
This line compares a variable to `None` which is not needed. You can simplify this by using `if 'vaccine' in visitor` instead of `if 'vaccine' not in visitor` and swapping the code in the `if` and `else` blocks.